### DISCUSSION TITLE
Pole Figure Angle Measurement

### DISCUSSION THREAD

**Discussion by t1ant12n** [mtex-toolbox/mtex#2204] [Ask Anything]
Hello,

I am writing to ask if MTEX has such a function to measure the angle between two poles on the pole figure. Like the one I did in aztec crystal (shown below).
![1](https://github.com/user-attachments/assets/08dd0f99-745f-4b2a-8b79-41b168e1f578)

Thanks in advance.

Tiantian

**Comment by lfgmorales** [comment]
Hi

A possibility:

First you need to calculate max, as here

<https://mtex-toolbox.github.io/S2Fun.max.html>
max | MTEX<https://mtex-toolbox.github.io/S2Fun.max.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/S2Fun.max.html>
        [favicon.png] <https://mtex-toolbox.github.io/S2Fun.max.html>



Then you can calculate angle as here

<https://mtex-toolbox.github.io/vector3d.angle.html>
angle | MTEX<https://mtex-toolbox.github.io/vector3d.angle.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/vector3d.angle.html>
        [favicon.png] <https://mtex-toolbox.github.io/vector3d.angle.html>


Cheers

Luiz

**Reply by t1ant12n** [reply]
Hello Luiz,

Thanks for your reply.

I think the **omega = angle(v1,v2)** gives the angle between two vectors. However, I want to decide the angle between two points on the pole figure. I think it is something related to the Wulff net.

Here is my code to get the plot. 

beta1 = find(job.parentGrains.id == 6684);
beta2=find(job.parentGrains.id == 6697);
oribeta1 = job.parentGrains(beta1).meanOrientation;
oribeta2 = job.parentGrains(beta2).meanOrientation;
h110=symmetrise(Miller(1,1,0,ebsd('Titanium cubic').CS));
figure
plot(oribeta1.* h110,'upper','MarkerFaceColor','b','label',char(h110,'cell'))
hold on
plot(oribeta2.* h110,'upper','MarkerFaceColor','r','label',char(h110,'cell'))

Here is the figure I got. 
<img width="783" alt="Screenshot 2024-09-01 at 11 13 59 AM" src="https://github.com/user-attachments/assets/928455ba-bc60-4474-88b7-4a9fd52b0f34">

So for example, is it possible to get the angle between the two points in the green circle in MTEX?

Thanks in advance.

Best regards,
Tiantian

**Reply by zmichels** [reply]
You can use angle() on vectors, but also on orientations. 

In a single-hemispherical plot, the points shown are vectors/directions. And if all you know are those directions, then that's what you have, and...
angle(v1, v2) will give you the angle between them.

If, alternatively, you have a complete orientational representation to work with related to each Miller direction/vector in that plot ... say o1, and o2... then you could use:
angle(o1, o2)
